src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/avoidescapedunicodecharacters/Example1.java

```diff
@@ -0,0 +1,23 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="AvoidEscapedUnicodeCharacters"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.avoidescapedunicodecharacters;
+
+// xdoc section -- start
+public class Example1 {
+  String unitAbbrev = "μs";     // OK, perfectly clear even without a comment.
+  //for the below line the reader has no idea what this is.
```

Comment:
This Check able to skip violation by trailing comments.
See allowByTailComment.

We should change all examples to avoid trailing comments with ok or violation.
We should bout them above a line.

All examples should be same by Java statements, and only comments are changing between ok and violation.

Trailing comments are only for one line `String unitAbbrev3 = "\u03bcs"; // it is  μs`. Please add this new statement to all examples, so user will see no violation if allowByTailComment=true .

Reply:
In `Example3.java` file when I added this line `String unitAbbrev3 = "\u03bcs"; // it is  μs` . There was no error but in all other files it says that it is unexpected. So you want me to change all files properties from `allowEscapesForControlCharacters` to `allowByTailComment`  ?

Error in Example1.java :  ```
[ERROR]   AvoidEscapedUnicodeCharactersCheckExamplesTest.testExample4:69->AbstractModuleTestSupport.verifyWithInlineConfigParser:238->AbstractModuleTestSupport.verifyViolations:468 Violation lines for C:\Users\r
ajro\checkstyle\src\xdocs-examples\resources\com\puppycrawl\tools\checkstyle\checks\avoidescapedunicodecharacters\Example4.java differ.
unexpected (1): 18
---
expected      : [17, 21]
but was       : [17, 18, 21]
```
```

/*xml
<module name="Checker">
  <module name="TreeWalker">
    <module name="AvoidEscapedUnicodeCharacters"/>
  </module>
</module>
*/

package com.puppycrawl.tools.checkstyle.checks.avoidescapedunicodecharacters;

// xdoc section -- start
public class Example1 {
  String unitAbbrev = "μs";     // OK, perfectly clear even without a comment.
  //for the below line the reader has no idea what this is.
  String unitAbbrev2 = "\u03bcs";// violation
  String unitAbbrev3 = "\u03bcs"; // it is  μs
  public static int content() {
    char content = 'r';
    return '\ufeff' + content;    //violation
  }
}
// xdoc section -- end

```

Reply:
Convert
`String unitAbbrev2 = "\u03bcs";// violation`
To
```
// violation below 
String unitAbbrev2 = "\u03bcs";
```

Di same for all others

Reply:
As we are using macroses, you do not need to share with me CLI out put, just simply change example, generate  xdoc files, commit and push it to GitHub.

Reply:
done with the changes review it once please @romani Sir